src/utils/teams.ts

```diff
@@ -78,6 +78,45 @@ export const getTeamHexColor = (color?: DomainColor) => {
   }
 };
 
+export const getTeamHexSecondaryColor = (color?: DomainColor) => {
+  switch (color) {
+    case DomainColor.Aqua:
+      return '--color-teams-yellow-200';
+    case DomainColor.Black:
+      return '--color-teams-red-200';
+    case DomainColor.Blue:
+      return '--color-teams-red-50';
```

Comment:
There must be a very valid reason for this but I just noticed that
- `DomainColor.Aqua` returns `yellow-200`
- `DomainColor.Blue` returns `red-50`

Reply:
Yes, the color stored for a domain has these `DomainColor` name even if visually it doesn't match with the displayed color.
I was initially tempted to update these utils functions from the `teams.ts` and soon realised the colors and no longer in sync 🫠